### PR TITLE
Rahil's updates of Cochabamba

### PR DESCRIPTION
**Pull request by RahilAlipour**
Updates and link to Wiki.

### CODE REVIEW

File: web_source/cities/south_america/bolivia/cochabamba.md
Comment (DeborahJohnstone) [inline web_source/cities/south_america/bolivia/cochabamba.md:1]: This looks great. I can't see the picture though - would you like to add one before I approve?

**Review by RahilAlipour — COMMENTED** [review]
all good :)

**Review by DeborahJohnstone — COMMENTED** [review]
Looks good, not sure if you want to add a picture?

**Review by DeborahJohnstone — APPROVED** [review]
Looks good!

**Pull request merged**